src/main/app/fact/FactApi.ts

```diff
@@ -62,10 +64,9 @@ export class FactApi {
       .get(`${this.adminBaseUrl}/openingTypes`)
       .then(results => results.data)
       .catch(err => {
-        this.logError(err);
+        this.errorHandler([]);
```

Comment:
This looks like an unintended change during merging.

Reply:
revert back to right implementation